package.json

```diff
@@ -40,6 +40,7 @@
     "react-dom": "^18.1.0"
   },
   "dependencies": {
+    "@kinde-oss/kinde-typescript-sdk": "^1.0.3",
```

Comment:
I released `1.0.4` earlier today, so worth updating to that to get the `uncrypto` changes.